### DISCUSSION TITLE
Exception during vertex undeploy step

### DISCUSSION THREAD

**Discussion by mariusz-wojcik-allegro** [microsoft/gctoolkit#311] [General]
Hi, after running gctoolkit application, I got some Exception:

![image](https://github.com/microsoft/gctoolkit/assets/88537233/6e987fe4-97b6-43ed-bde2-68bf5babf32c)

Have you ever seen it ? How can I fix it ?

Best regards
mario

**Comment by kcpeppe** [comment]
yeah, I thought we'd gotten rid of it. It's a benign annoyance that can safely be ignored. Its caused when more than 1 parser shuts down vert.x. I've just started a bug fixing run... I'll put it into the list to see why it's come back.

**Reply by karianna** [reply]
I think the move to vert.x 4.5.0 has bought it back again.
